README.md

```diff
@@ -257,6 +257,7 @@ Questions can be asked by raising an `Issue`.
 - [Remco Halman](https://github.com/remcohalman)
 - [Richard Bacon](https://github.com/RichardJonBacon)
 - [Rick Flores](https://github.com/RICK-FLORES)
+- [Rutik Bhoyar](https://github.com/Rutikab12)
```

Comment:
Could you add yourself in alphabetical order? After your name there are a some people with `Ri`.

Reply:
okay sir